examples/javascript/src/migration-lambda/index.mjs

```diff
@@ -75,3 +88,17 @@ export const handler = async (event) => {
 
   return event;
 };
+
+const addAreaCodeToPhone = (phone) => {
+  // Has 11 digits (has country code), but missing +
```

Comment:
i think these are the cases we have to handle:

* number has no leading `+` <- add `+` to beginning
* number has no country code: `+3334445555` <- insert a `1` after the plus
* number has no leading `+` or country code: `3334445555` <- add `+1` to beginning

in the second case, this won't catch that it's a 10-digit because it's got a leading `+`.

dumb thought. we could do something like:

* remove leading plus
* if it's a 10-digit, we can fix: add a `1` to beginning
* shift the `+` back on the front